traffic_ops/app/db/migrations/20200805000000_add_deliveryservice_service_category.sql

```diff
@@ -0,0 +1,25 @@
+/*
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+		http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+CREATE TABLE IF NOT EXISTS service_category (
```

Comment:
this needs to be renamed to `2020080500000000_add_deliveryservice_service_category.sql`(add 2 zeros) to ensure this is the last migration in the list.

Reply:
I didn't think this would matter because the alphabetic sort order was unchanged, but it looks like goose first sorts by string length, then alphabetically - for some reason:

```psql
traffic_ops=> SELECT * FROM goose_db_version;
 id |    version_id    | is_applied |           tstamp           
----+------------------+------------+----------------------------
  1 |                0 | t          | 2020-08-06 23:49:38.325314
  2 |   20181206000000 | t          | 2020-08-06 23:49:38.331942
  3 |   20190219000000 | t          | 2020-08-06 23:49:38.333987
  4 |   20190319000000 | t          | 2020-08-06 23:49:38.335588
  5 |   20190513000000 | t          | 2020-08-06 23:49:38.36628
  6 |   20191004000000 | t          | 2020-08-06 23:49:38.375986
  7 |   20191005000000 | t          | 2020-08-06 23:49:38.384214
  8 |   20191024000000 | t          | 2020-08-06 23:49:38.39299
  9 |   20191215000000 | t          | 2020-08-06 23:49:38.401664
 10 |   20200218000000 | t          | 2020-08-06 23:49:38.432109
 11 |   20200227000000 | t          | 2020-08-06 23:49:38.437236
 12 |   20200313000000 | t          | 2020-08-06 23:49:38.439051
 13 |   20200408000000 | t          | 2020-08-06 23:49:38.516281
 14 |   20200422101648 | t          | 2020-08-06 23:49:38.524859
 15 |   20200507000000 | t          | 2020-08-06 23:49:38.569508
 16 |   20200805000000 | t          | 2020-08-06 23:49:38.588792
 17 | 2020061622101648 | t          | 2020-08-06 23:49:38.598169
 18 | 2020062923101648 | t          | 2020-08-06 23:49:38.600003
 19 | 2020072700000000 | t          | 2020-08-06 23:49:38.909138
(19 rows)
```